MicrowaveGame/Assets/Resources/Scripts/Dialogue/DialogueBehaviour.cs

```diff
@@ -0,0 +1,86 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class DialogueBehaviour : MonoBehaviour
```

Comment:
This should be in the namespace `namespace Scripts.Dialogue { ... }`.

Reply:
implementing the namespace created an issue where the code wants "namespace.DialogueTrigger" when using GetComponent<>. Is that supposed to happen, because I haven't seen that syntax in other scripts from what I've seen

Reply:
Hmmm, there shouldn't be an issue if all three dialogue classes (`Dialogue`, `DialogueBehaviour`, and `DialogueTrigger`) are in the same namespace (`namespace Scripts.Dialogue`).

Reply:
Seems I had to include a "using Scripts.DialogueUI" line. interesting

Reply:
For future reference, try to match the namespace to the folder structure. In this case, that would be `Scripts.Dialogue`.